src/visualization/write2vtk.jl

```diff
@@ -384,6 +384,9 @@ function write2vtk!(vtk, v, u, t, model::BoundaryModelDummyParticles, system;
 
     if model.viscosity isa ViscosityAdami
         vtk["wall_velocity"] = view(model.cache.wall_velocity, 1:ndims(system), :)
+    else
+        vtk["velocity"] = [current_velocity(v, system, particle)
+                           for particle in active_particles(system)]
```

Comment:
I have the velocity written out for 'boundary_systems' instead of the dimension of the system. The 'meta_data' is only written out in a simulation and not in individually created systems (such as in the example file readvtk.jl). This makes it possible in any case to adapt the size of the coordinate array to the size of the velocity array.